src/ui/public/vislib/visualizations/point_series/line_chart.js

```diff
@@ -94,7 +96,8 @@ export default function LineChartFactory(Private) {
           const height = self.baseChart.chartConfig.height;
           const circleRadius = (d.z - radii.min) / radiusStep;
 
-          return _.min([Math.sqrt((circleRadius || 2) + 2), width, height]) + (modifier || 0);
+          const base = circleRadius ? Math.sqrt(circleRadius + 2) + lineWidth : lineWidth;
```

Comment:
When trying it out I found it confusing that the `lineWidth` influences the dot size even if I specified a dot size metric. Shouldn't the `lineWidth` be ignored when calculating the circle size in that case?